airbyte-integrations/bases/base-normalization/integration_tests/test_normalization.py

```diff
@@ -332,6 +389,31 @@ def setup_dbt_incremental_test(destination_type: DestinationType, test_resource_
     )
 
 
+def setup_dbt_sparse_nested_streams_test(destination_type: DestinationType, test_resource_name: str, test_root_dir: str, sync_number: int):
+    """
+    Prepare the data (copy) for the models for dbt test.
+    """
+    replace_identifiers = os.path.join("resources", test_resource_name, "data_input", "replace_identifiers.json")
```

Comment:
since `replace_identifiers.json` does not have any replacements, you dan skip it altogether, because `copy_test_files` will ignore it if the file does not exist:
```
if os.path.exists(replace_identifiers):
...
```